README.rst

```diff
@@ -593,6 +593,216 @@ For example, if we want to change the verbose name to "Hotspot", we could write:
 
     OPENWISP_CONTROLLER_DEVICE_VERBOSE_NAME = ('Hotspot', 'Hotspots')
 
+``OPENWISP_CONTROLLER_API``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+-----------+
+| **type**:    | ``bool``  |
++--------------+-----------+
+| **default**: | ``True``  |
++--------------+-----------+
+
+Indicates whether the API for Openwisp Controller is enabled or not.
+
+REST API
+--------
+
+To enable the API the setting `OPENWISP_CONTROLLER_API <#openwisp-controller-api>`_
+must be set to ``True``.
+
+Live documentation
+~~~~~~~~~~~~~~~~~~
+
+.. image:: https://raw.githubusercontent.com/ManishShah120/openwisp-controller/issues/379-REST_API-for-main-controller/docs/live-docu-api.png
```

Comment:
reminder that we'll have to update these screenshots to point to master once we're ready to merge

Reply:
Sure, Once we are ready to merge, I'll Update it.